### ISSUE TITLE
[Questions]

### ISSUE THREAD

**Issue by Jorman** [isc30/eMulerr#8]
Hi, I just discovered this work and I must say it is very interesting, it adds a very good possibility to all *Arr, great job!

I have just a couple of questions/curiosities to ask you:
- I saw that you already have all the ngosang patches, but the docker configuration looks slightly different to me. Is it possible to access the aMule web interface? I have tried via 4711, I get to the log-in screen, the classic aMule webui screen, but whatever password I enter won't let me log in. I'm asking because I would like to pause downloads and haven't found how else to do it. I would also like to see the "Last seen Complete" status which I can't always see on the standard interface that runs on the 3000
- Have you ever thought of doing the same thing but for the XDCC network as well? I was just looking for that before I found this project of yours and I have to say that it would be the best to have that network connected directly to the *Arr as well, just like you did with ed2k! There are engines like https://www.xdcc.eu/ that provide good results, I don't know of any clients or automatons to do that though. Do you have any advice on this?

Thank you
J

**Comment by isc30** [comment]
Hi, thanks for the nice feedback!!

1. If you want to access the amule web interface you can expose the port 4711 and use the built-in password `secret`:
  ```diff
    ports:
+   - "4711:4711" # amule webui
   ```

2. No plans to implement XDCC, will have a look tho as I'm not familiar with it :)

**Comment by Jorman** [comment]
Thanks @isc30 for the reply

Indeed now I can connect to the "normal" amule interface, sometimes it is necessary!

For XDCC or more generally DC++ is a great network, maybe the best "I'm old" when I was young I used to download via radio waves!
If and when you want to have a look at the DC++ network, you will see that there is a lot of material it is a great protocol, too bad it lacks the possibility to add it to our dear `*Arr`
Ps, just for your information, in the meantime I did research and found that `AirDCPP` allows plug-ins, but still no one has ever created one for integration.

Just to "complete" the topic

Thanks again for the login password, and for `eMulerr`!
